src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/decommission-status.go

```diff
@@ -0,0 +1,110 @@
+package brokers
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/types"
+)
+
+func newDecommissionBrokerStatus(fs afero.Fs) *cobra.Command {
+	var (
+		completion int
+		detailed   bool
+	)
+	cmd := &cobra.Command{
+		Use:   "decommission-status [BROKER ID]",
+		Short: "Show the progress of a node decommissioning",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			broker, _ := strconv.Atoi(args[0])
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			dbs, err := cl.DecommissionBrokerStatus(cmd.Context(), broker)
+			if he := (*admin.HTTPResponseError)(nil); errors.As(err, &he) {
+				// Special case 400 (validation) errors with friendly output
+				// about the node is not decommissioning
+				if he.Response.StatusCode == 400 {
+					body, bodyErr := he.DecodeGenericErrorBody()
+					if bodyErr == nil {
+						out.Exit("%s", body.Message)
+					}
+				}
+			}
+			out.MaybeDie(err, "unable to request brokers: %v", err)
+
+			if dbs.Finished {
+				if dbs.ReplicasLeft == 0 {
+					out.Exit("Note %d is decommissioned successfully.", broker)
+				} else {
+					out.Exit("Node %d is decommissioned but %d replicas left. So that there is likely an issue inside Redpanda. Please describe how you encountered this at https://github.com/redpanda-data/redpanda/issues/new?assignees=&labels=kind%2Fbug&template=01_bug_report.md", broker, dbs.ReplicasLeft)
```

Comment:
The wording we use in config/params.go is:
```go
"rpk bug, please describe how you encountered this at https://github.com/redpanda-data/redpanda/issues/new?assignees=&labels=kind%2Fbug&template=01_bug_report.md"
```

wdyt about:
```go
out.Exit("Node %d is decommissioned but there are %d replicas left, which may be an issue inside Redpanda. Please describe how you encountered this at https://github.com/redpanda-data/redpanda/issues/new?assignees=&labels=kind%2Fbug&template=01_bug_report.md")
```

Reply:
Makes sense. Thank you.